src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -49,7 +49,7 @@ internal static class FastTreeShared
     public abstract class FastTreeTrainerBase<TArgs, TTransformer, TModel> :
         TrainerEstimatorBaseWithGroupId<TTransformer, TModel>
         where TTransformer : ISingleFeaturePredictionTransformer<TModel>
-        where TArgs : TreeArgs, new()
+        where TArgs : TreeOptions, new()
         where TModel : class
     {
         protected readonly TArgs Args;
```

Comment:
>TArgs Args [](start = 27, length = 10)

TOptions Options;

Reply:
Options cause compile errors because of conflict with class Options. Renamed to FastTreeTrainerOptions

---
In reply to: [257077340](https://github.com/dotnet/machinelearning/pull/2563#discussion_r257077340) [](ancestors = 257077340)